src/main/java/com/mycompany/myapp/web/rest/reservation/ReservationResource.java

```diff
@@ -40,4 +40,11 @@ public ReservationResource(ReservationService reservationService){
         reservationService.deleteReservation(id);
         return ResponseEntity.ok().headers(HeaderUtil.createAlert( "reservation.deleted", id.toString())).build();
     }
+
+    @GetMapping("/request-tools")
```

Comment:
Why you don't want to make it a POST request and send all necessary data in request body?